src/Fantomas.Core.Tests/TypeAnnotationTests.fs

```diff
@@ -48,14 +48,12 @@ type Meh
         equal
         """
 type Meh
-    (
-        input:
-            LongTupleItemTypeOneThing *
-            LongTupleItemTypeThingTwo *
-            LongTupleItemTypeThree *
-            LongThingFour *
-            LongThingFiveYow
-    ) = class end
+    (input:
+        LongTupleItemTypeOneThing *
+        LongTupleItemTypeThingTwo *
+        LongTupleItemTypeThree *
+        LongThingFour *
+        LongThingFiveYow) = class end
 """
```

Comment:
This never had anything to do with the `MultilineBracketStyle` setting.
See https://learn.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-function-and-member-arguments

Reply:
What am I missing here, it looks like this change moves away from what's in the style guide:

```fsharp

// ✔️ OK
type TypeWithLongConstructor
    (
        aVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
        aSecondVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
        aThirdVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse
    ) =
    // ... the body of the class follows

// ✔️ OK
type TypeWithLongSecondaryConstructor () =
    new
        (
            aVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
            aSecondVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
            aThirdVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse
        ) =
        // ... the body of the constructor follows
```

Reply:
That is because this isn't a tuple. This is a single parameter that happens to have a long multiline tuple type. So it shouldn't follow the tuple guidance in the guide as it is not a tuple.

Reply:
Ah I see. It feels kind of odd if it's the only param bc I would personally expect it to match the normal style, but if that would require a bunch of weird handling I think I'm fine with it

Reply:
Hmm, yeah maybe this should be changed. It does deviate from what was originally mentioned in https://github.com/dotnet/docs/pull/31499/files.
I'm gonna take a quick peek, if it is too much hassle I might skip it thought.

Reply:
@josh-degraw ok, it wasn't that bad, lemme know what you think.

Reply:
@josh-degraw Can you please respond in literally any other way than the 👀 button here? 🫠

I'm looking for your approval here if you are behind this change.
The eyes thing doesn't notify me and does not explicitly tell me whether this is good to go or not.

Reply:
So sorry! I was looking to see what changes you'd made but I'm not seeing anything new since my last review, did you push up the changes?

Reply:
Well, this is embarrassing, I indeed didn't push these changes.

Reply:
Turns out I did push but to another branch. @josh-degraw pushed here as well now.

Reply:
Awesome, yeah that looks much more like what I would expect